prowler/providers/aws/services/sqs/sqs_service.py

```diff
@@ -49,18 +49,25 @@ def __list_queues__(self, regional_client):
 
     def __get_queue_attributes__(self, regional_clients):
         try:
+            logger.info("SQS - describing queue attributes...")
             for queue in self.queues:
                 regional_client = regional_clients[queue.region]
                 queue_attributes = regional_client.get_queue_attributes(
-                    QueueUrl=queue.id
+                    QueueUrl=queue.id, AttributeNames=["All"]
                 )
-                if (
-                    "Attributes" in queue_attributes
-                    and "Policy" in queue_attributes["Attributes"]
-                ):
-                    queue.policy = loads(queue_attributes["Attributes"]["Policy"])
-                if "KmsMasterKeyId" in queue_attributes["Attributes"]:
-                    queue.kms_key_id = queue_attributes["Attributes"]["KmsMasterKeyId"]
+                if "Attributes" in queue_attributes:
+                    if "Policy" in queue_attributes["Attributes"]:
+                        queue.policy = loads(queue_attributes["Attributes"]["Policy"])
+                    if "KmsMasterKeyId" in queue_attributes["Attributes"]:
+                        queue.kms_key_id = queue_attributes["Attributes"][
+                            "KmsMasterKeyId"
+                        ]
+                    if "SqsManagedSseEnabled" in queue_attributes["Attributes"]:
+                        if (
+                            queue_attributes["Attributes"]["SqsManagedSseEnabled"]
+                            == "true"
+                        ):
+                            queue.kms_key_id = "SqsManagedSseEnabled"
```

Comment:
Please, include the tests to cover this behaviour.

Reply:
The function GetAttributes is mocked, so we cannot create different use cases.